### ISSUE TITLE
target="_blank" rel="noreferrer" implies noopener

### ISSUE THREAD

**Issue by simonhaenisch** [Microsoft/tslint-microsoft-contrib#571]
```html
<a href="#" target="_blank" rel="noreferrer"></a>
```

> [tslint] Anchor tags with target="_blank" should also include rel="noopener noreferrer"

But, as pointed out here: https://twitter.com/_developit/status/1054422343609655296

> `<a rel="noreferrer">` implies noopener. Specifying both is redundant.

https://github.com/yannickcr/eslint-plugin-react/issues/2022

**Comment by JoshuaKGoldberg** [comment]
Thanks for pointing this out @simonhaenisch!

As ljharb mentioned in the eslint-plugin-react reference, it's unclear which browsers support that behavior. It wouldn't be surprising if IE behaved wrongly here. We could add an option for this rule to allow not enforcing `noopener` if `noreferrer` is there.
